php/libraries/NDB_Client.class.inc

```diff
@@ -116,7 +116,8 @@ class NDB_Client
         header(
             "Content-Security-Policy: "
             . "default-src 'self' 'unsafe-inline'; "
-            . "script-src 'self' 'unsafe-inline' 'unsafe-eval'; "
+            . "script-src 'self' 'unsafe-inline' 'unsafe-eval'"
+            . "www.google.com  www.gstatic.com; "
```

Comment:
this should have an if statement too, if the captcha isn't set up, we shouldn't make the exception.